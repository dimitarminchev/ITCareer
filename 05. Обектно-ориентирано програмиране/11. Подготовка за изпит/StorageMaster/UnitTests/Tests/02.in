AddProduct HardDrive -20
AddProduct Product 20
RegisterStorage InvalidStorage LoshHackerStorage
RegisterStorage Warehouse GoodHackerStorage
SelectVehicle GoodHackerStorage 0
LoadVehicle HardDrive
SendVehicleTo LoshHackerStorage 0 GoodHackerStorage
SendVehicleTo GoodHackerStorage 0 LoshHackerStorage
END
