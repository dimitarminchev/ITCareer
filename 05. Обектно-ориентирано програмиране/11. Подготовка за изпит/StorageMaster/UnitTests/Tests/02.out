Error: Price cannot be negative!
Error: Invalid product type!
Error: Invalid storage type!
Registered GoodHackerStorage
Selected Semi
Error: HardDrive is out of stock!
Error: Invalid source storage!
Error: Invalid destination storage!
GoodHackerStorage:
Storage worth: $0.00
